pkg/config/oc/default.go

```diff
@@ -516,13 +516,12 @@ func OverwriteNeighborConfigWithPeerGroup(c *Neighbor, pg *PeerGroup) error {
 
 func overwriteConfig(c, pg interface{}, tagPrefix string, v *viper.Viper) {
 	nValue := reflect.Indirect(reflect.ValueOf(c))
-	nType := reflect.Indirect(nValue).Type()
 	pgValue := reflect.Indirect(reflect.ValueOf(pg))
```

Comment:
The changes to this file are related with this pull request?

Reply:
Since I removed `bgp_configs.go` change, no, they're no longer are. I put it into separate PR - https://github.com/osrg/gobgp/pull/2873
Thanks.